### ISSUE TITLE
Fieldsets option doesn't show on reply if there are no record-level ticket custom fields

### ISSUE THREAD

**Issue by Hildy** [jstanden/cerb#1286]
If you don't have any global ticket custom fields, then the fieldsets option doesn't show up on ticket Reply.

**Comment by jstanden** [comment]
~~~
Index: features/cerberusweb.core/templates/display/rpc/reply.tpl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- features/cerberusweb.core/templates/display/rpc/reply.tpl	(revision e3b23bcf36e47df8eabf26644c4747cfac0e760f)
+++ features/cerberusweb.core/templates/display/rpc/reply.tpl	(date 1588701387175)
@@ -300,7 +300,7 @@
 
 {$custom_fieldsets_available = DAO_CustomFieldset::getUsableByActorByContext($active_worker, CerberusContexts::CONTEXT_TICKET)}
 
-{if $custom_fields || $custom_fields_available}
+{if $custom_fields || $custom_fieldsets_available}
 <fieldset class="peek" style="{if $custom_fieldsets_available}padding-bottom:0px;{/if}">
 	<legend>
 		<label>
~~~

**Comment by jstanden** [comment]
Fixed in 9.5.6
